Added link to the episode on version control (#53)
carpentries-incubator/fair-research-software: episodes/02-tools.md

::::::::::::::::::::::::::::::::::::::::::::::::

> You may have noticed that our reseacher has received the software project they are meant to be working as a `.zip` archive that was emailed to them. 

Author: ```suggestion
You may have noticed that our researcher has received the software project they are meant to be working as a `.zip` archive via email. 
```

In the next episode, we will learn a better practice for sharing and tracking changes to a software project using version control software Git and 

> project sharring and collaboration platform GitHub.

Reviewer: ```suggestion
project sharing and collaboration platform GitHub.
```
